teuthology/openstack/__init__.py

```diff
@@ -1048,6 +1049,9 @@ def get_user_data(self):
             all_options += [ '--network ' + network ]
         if self.args.simultaneous_jobs:
             all_options += [ '--nworkers ' + str(self.args.simultaneous_jobs) ]
+        if self.args.nameserver:
+            all_options += [ '--nameserver %s' % self.args.nameserver]
+
```

Comment:
don't we need to remove `--nameserver` from the arguments that are passed to `teuthology-suite` ?

Reply:
nvm